### ISSUE TITLE
Objects set with the "Draw" checkbox un-checked (in Materials) still draw.

### ISSUE THREAD

**Issue by danklaue** [der-On/XPlane2Blender#259]
For manipulators, we need the objects we model, to not draw, but still be there and clickable.  For that reason, there's been a "Draw" checkbox in the "Materials" menu under the X-Plan tab:

<img width="505" alt="screen shot 2017-06-05 at 6 19 16 pm" src="https://cloud.githubusercontent.com/assets/359923/26807350/9c2925d0-4a1b-11e7-88e3-fcfa615bbf17.png">

This used to work fine in previous versions of the plugin.  It is no longer working in 3.3.13.  It is exporting manipulators (set to NOT draw) as visible objects in X-Plane.

**Comment by danklaue** [comment]
Noticing that it has something to do with the "AircraftName_cockpit.png" texture. If the manipulator object has been (inadvertently) UV-mapped with that texture, it'll display visibly in X-Plane, despite having been set to not be visible.  Of course, clearing the "cockpit.png" texture (making it completely transparent) causes the manipulators to indeed vanish in X-Plane.  

The problem remains: objects that were set to invisible exported invisible in older versions of the export script. Now, they no longer do so.

**Comment by bsupnik** [comment]
Hi Dan,

I need a small .blend test case that reproduces this.

**Comment by danklaue** [comment]
OK, it seems like when I tried to create a test file, everything worked as it should. This leads me to conclude that it might be a termination issue.  I'll try to create a sample of that later.

But here's what I've seen in the past: You have a cockpit layer, with some objects set to be invisible manipulators, others set to be part of the panel, and yet others, set to be regular textured objects (textured with the "*_cockpit.png" file.)

You export it once, everything seems to work fine.  Then you tweak the object that's part of the cockpit panel, and re-export it.  Suddenly, all the manipulators show up.  To solve this, I've made a duplicate of the manipulator objects, and deleted the original. What this does is, it exports the part of interest higher up in the hierarchy, and allows its attributes to be valid.  

It's probably just a matter of checking if the attribute of visibility is initiated and terminated properly.

But I'll try to create a sample file later on today.

**Comment by danklaue** [comment]
OK, I got a test case here:

[RV-10_TestPlane.zip](https://github.com/der-On/XPlane2Blender/files/1056292/RV-10_TestPlane.zip)

There's definitely a bug here somewhere, but it's really difficult to figure out what's going on.

<img width="963" alt="screen shot 2017-06-06 at 3 30 00 pm" src="https://user-images.githubusercontent.com/359923/26850539-7a253b56-4acd-11e7-80ee-168fedc5769c.png">


I narrowed down the scene to only 4 objects:

1. A manipulator, set to "NOT DRAW". (Hooked up to Parking Brakes.)

2. A regular object, set up as though it were a gauge. Responds to Parking Brake dataref with rotation.  Set to visible.

3. Another regular object, with a different material.  Set to visible.

4. another manipulator, parented to regular object 3. Set to invisible.

Exporting this file as-is, will cause object 1 to be visible in X-Plane. (I.e., "DON'T DRAW" setting is not respected.)

However, doing any of the following, will cause it to export correctly:

1. Un-parent object 4.
2. hide or delete object 3 or 4.
3. Set object 2's material to be the same as object 3's material.
